src/App/WarehouseService.cs

```diff
@@ -3,25 +3,45 @@
 
 public class WarehouseService
 {
-    private readonly string _connectionString = "Host=localhost;Username=postgres;Password=yourpassword;Database=warehouse";
+    private readonly string _connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
 
```

Comment:
add this line for logging private static readonly log4net.ILog log = log4net.LogManager.GetLogger(typeof(WarehouseService));